mne/simulation/tests/test_metrics.py

```diff
@@ -1,50 +0,0 @@
-# Author: Yousra Bekhti <yousra.bekhti@gmail.com>
-#         Mark Wronkiewicz <wronk@uw.edu>
-#
-# License: BSD (3-clause)
-
-
-import os.path as op
-
-import numpy as np
-from numpy.testing import assert_almost_equal
-import pytest
-
-from mne import read_source_spaces
-from mne.datasets import testing
-from mne.simulation import simulate_sparse_stc, source_estimate_quantification
```

Comment:
@makkostya by changing:

`from mne.simulation import simulate_sparse_stc, source_estimate_quantification`
to
`from mne.simulation.metrics import simulate_sparse_stc, source_estimate_quantification`

you broke backward compatibility.

you need to revert this change and deprecate the functions that have now a new alternative API.
Do we deprecate source_estimate_quantification only or simulate_sparse_stc too?

Reply:
> you need to revert this change

How can I do it without loosing all other changes? I have never done revert before.

> Do we deprecate source_estimate_quantification only or simulate_sparse_stc too?

The backward compatibility of simulate_sparse_stc is not affected.